### DISCUSSION TITLE
help me name a movie!

### DISCUSSION THREAD

**Discussion by tattiyaS** [NAIST-SE/AttractingContributors#43] [General]
url: https://github.com/tattiyaS/AttractingContributors

**Comment by vittunyuta-aom** [comment]
added!!

**Reply by tattiyaS** [reply]
thank you

**Comment by mailtoy** [comment]
I added! check it ;-;

**Reply by tattiyaS** [reply]
yay!

**Comment by haxhimitsu** [comment]
I already added your repo!

Thanks

**Reply by tattiyaS** [reply]
thank you!

**Comment by satokibi44** [comment]
added!

**Reply by tattiyaS** [reply]
thank you very much!

**Comment by wowry** [comment]
I have sent a PR! :)

**Reply by tattiyaS** [reply]
thank you

**Comment by 278Mt** [comment]
Added! and plz help me https://github.com/278Mt/AttractingContributors
thx;-)

**Reply by tattiyaS** [reply]
thank you

**Comment by eedy8888** [comment]
added, please help me
https://github.com/eedy8888/AttractingContributors

**Reply by tattiyaS** [reply]
done!

**Comment by manzoku23** [comment]
Added my favorite one.
Please help me 
https://github.com/Exe-dev/AttractingContributors-POKEMON

**Comment by sergiodl-naist** [comment]
added one more!
Please visit and try to ask a question or recommend one vtuber on my fork!
https://github.com/sergiodl-naist/AttractingContributors-VTubers
